MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/Muon/MuonAnalysis.h

```diff
@@ -249,6 +249,9 @@ private slots:
   /// Called when "enable multi fit" checkbox is turned on/off
   void multiFitCheckboxChanged(int state);
 
+  /// Called when "TF Asymmetry" checkbox is turned on/off
+  void TFAsymmCheckboxChanged(int state);
```

Comment:
I understand, but the convention is to have the starting letters small case, one reason off the top of my head is that future developers will not know about the capital T, and the autocomplete will not help them with the function name (I've had this happen, it's annoying to have to dig through the source to find that the letter is capital)